### PR TITLE
register insight

### PR DESCRIPTION
**Pull request by jburel**
Insight was no longer registered

* If you use ``gradle run`` an entry with ``Unknown`` for the version will be added, otherwise it will be ``5.5.2-SNAPSHOT`` 

To test:
* Connect to merge-ci server
* Check that insight is registered
  * Go to ``omero-web-prod``
  * check the ``upgrade access`` log

cc @sbesson

### CODE REVIEW

**Comment by mtbc** [comment]
With `gradle run` from my laptop recorded as,

> 10.50.24.139 - - [11/Jun/2019:15:18:56 +0100] "GET /?version=Unknown;os.name=Linux;os.arch=amd64;os.version=4.9.0-9-amd64;java.runtime.version=1.8.0_212-8u212-b03-2%7Edeb9u1-b03;java.vm.vendor=Oracle+Corporation HTTP/1.1" 301 507 "-" "OMERO.insight"

**Comment by jburel** [comment]
@mtbc thanks

**Comment by mtbc** [comment]
Today's merge-ci build registers as,

> 10.50.24.139 - - [12/Jun/2019:09:19:29 +0100] "GET /?version=5.5.2-SNAPSHOT;os.name=Linux;os.arch=amd64;os.version=4.9.0-9-amd64;java.runtime.version=1.8.0_212-8u212-b03-2%7Edeb9u1-b03;java.vm.vendor=Oracle+Corporation HTTP/1.1" 301 514 "-" "OMERO.insight"

File: src/main/java/org/openmicroscopy/shoola/env/data/DataServicesFactory.java [hunk 1]
```diff
@@ -516,6 +522,14 @@ public void connect(UserCredentials uc)
             registry.getLogger().warn(this, "Could not access ConfigService");
         }
 
+        //Register insight
+        try {
+            UpgradeCheck check = new UpgradeCheck(cs.getConfigValue("omero.upgrades.url"), clientVersion, name);
+            check.run();
+        } catch (ServerError e2) {
+            registry.getLogger().warn(this, "Could not access ConfigService");
```
Comment (mtbc) [inline src/main/java/org/openmicroscopy/shoola/env/data/DataServicesFactory.java]: maybe could include cause (last `warn` arg of `e2`)

*Not* a blocker, can revisit post-release.

Comment (jburel) [reply src/main/java/org/openmicroscopy/shoola/env/data/DataServicesFactory.java]: We have few cases like that in this class. I will review post release

**Pull request merged**